alaska/tests/test_parser.py

```diff
@@ -42,6 +64,50 @@ def test_parse():  # 1000080059
     assert result == ({"depth": ["DEPT"], "gamma ray": ["GR"]}, [])
 
 
+def test_parse_2():
+    """
+    Test that Aliaser can parse las file with an empty mnemonic
+    """
+    aliaser = Alias()
+    # import pdb; pdb.set_trace()
+    result = aliaser.parse(test_case_5)
+    assert result == ({"depth": ["DEPT"], "gamma ray": ["GR"]}, ["empty"])
+
+
+def test_parse_directory_1():
+    """
+    Test that Aliaser can parse a directory of las files
+    """
+    aliaser = Alias()
+    aliased, not_aliased = aliaser.parse_directory(test_dir_1)
+    have1 = aliased.get("density porosity", None)
+    have2 = aliased.get("depth", None)
+    have3 = aliased.get("gamma ray", None)
+
+    # Aliased
+    assert have1 == ["DPHI"]
+    assert have2 == ["DEPT"]
+    assert have3 == ["GR"]
+
+    # Not Aliased
+    assert "qn" in not_aliased
+    assert "empty" in not_aliased
+
+
```

Comment:
I think the `model` only parsing test is really useful to have. Nice job on these tests!

Reply:
Great, I glad these are useful. :-)